src/Nokia_LCD.h

```diff
@@ -182,6 +188,24 @@ class Nokia_LCD {
      */
     bool draw(const unsigned char bitmap[], const unsigned int bitmap_size,
               const bool read_from_progmem = true);
+    /**
+     * Draws the supplied bitmap on the screen starting at the current cursor
+     * location. The bitmap can contain up to 504 bits which is the amount of
+     * pixels in the display. When bitmap is smaller than screen width, is 
+     * necessary to inform the width.
+     *  
+     * @param  bitmap            The bitmap to be displayed
+     * @param  bitmap_size       The size of the bitmap to be displayed up to
+     *                           504 bits
+     * @param  bitmap_width      The bitmap width. By default uses the screen width.
+     * @param read_from_progmem  Whether the bitmap is stored in flash memory
+     *                           instead of SRAM. Default read from flash.
+     * @return                   True if out of bounds error | False otherwise
+     */
+    bool draw(const unsigned char bitmap[],
+              const unsigned int bitmap_size,
+              const unsigned int bitmap_width = kDisplay_max_width,
+              const bool read_from_progmem = true);
```

Comment:
I'm a bit surprised this actually compiles. If two arguments are supplied, how does the compiler know which `draw` to use? This or the one further up?

In any case, since you added the default arguments, you don't need both functions anymore. You can safely delete the old one and only keep the new version with the four arguments.

Reply:
Well, after some tests, I would say the compiler actually do not know. It can't handle these default params very well. I removed one of the draw methods but it doesn't work. 
For example, when I call `lcd.draw(platis_solutions_logo, 504, true);` the default bitmap width should be 84 but it is 1! The compiler is casting the last param true to bitmap_width.

It makes me think one thing, the bitmap_width param shouldn't have default value since we are providing a method draw() without bitmap_width for that case. If user don't want to pass bitmap_width, call the original draw(). So, I'm keeping both draw() methods, I don't want to break compatibility.

Reply:
OK, you are right, it's a good idea to keep backwards compatibility. In that case, we can simply move the `bitmap_width` argument last. This way all previous code is compatible with the newer version.

As I mentioned, I very surprised it compiles for Arduino. When I try out the following simple example on my machine, it doesn't compile as expected because the call is ambiguous.

```cpp
bool draw(const unsigned char bitmap[],
          const unsigned int bitmap_size,
          const bool read_from_progmem = true)
{
    return false;
}

bool draw(const unsigned char bitmap[],
          const unsigned int bitmap_size,
          const unsigned int bitmap_width = 5,
          const bool read_from_progmem    = true)
{
    return true;
}

int main(int const argc, char const* const* const argv)
{
    const unsigned char bitmap[] = { 1, 2, 3 };
    draw(bitmap, 3);

    return 0;
}
```

> /home/user/CLionProjects/TryOuts/main.cpp:241:19: error: call of overloaded ‘draw(const unsigned char [3], int)’ is ambiguous
  241 |     draw(bitmap, 3);
      |                   ^
/home/user/CLionProjects/TryOuts/main.cpp:223:6: note: candidate: ‘bool draw(const unsigned char*, unsigned int, bool)’
  223 | bool draw(const unsigned char bitmap[],

Reply:
Nice, moving bitmap_width to the last param solved the issue I had. I'm pushing the changes